Postman Test Suite/Ed-Fi ODS-API Profile Test Suite.postman_collection.json

```diff
@@ -9539,6 +9655,111 @@
 								}
 							},
 							"response": []
+						},
+						{
+							"name": "The Write content type includes all properties of a  derived association",
+							"event": [
+								{
+									"listen": "test",
+									"script": {
+										"exec": [
+											"pm.test(\"The response should indicate success\", () => {\r",
+											"    pm.expect(pm.response.code).to.equal(204);\r",
+											"});\r",
+											"\r",
+											"// Made it a function, to avoid async issues\r",
+											"function getKnownSchool (callback) {\r",
+											"    pm.sendRequest({\r",
+											"        url: pm.environment.get(\"ApiBaseUrl\") + `/data/v3/ed-fi/studentSpecialEducationProgramAssociations/` + pm.environment.get(\"known:studentSpecialEducationProgramAssociationId\"),\r",
+											"        method: \"GET\",\r",
+											"        header: {\r",
+											"            \"Authorization\": \"bearer \" + pm.environment.get(\"AccessToken_255901\"),\r",
+											"            \"Content-Type\": \"application/json\",\r",
+											"        }\r",
+											"    }, callback);\r",
+											"}\r",
+											"\r",
+											"getKnownSchool((error, response) => {\r",
+											"    Validate(response.json());\r",
+											"});\r",
+											"\r",
+											"function Validate(jsonResponse) {\r",
+											"    const originalEntity = pm.environment.get(\"known:studentSpecialEducationProgramAssociation\");\r",
+											"    const updatedEntity = jsonResponse;\r",
+											"    const keys = pm.environment.get(\"known:studentSpecialEducationProgramAssociations:PrimaryKeys\");\r",
+											"\r",
+											"    pm.test(\"The persisted entity model should have unmodified primary key values\",\r",
+											"        () => {\r",
+											"            _.each(keys, (key) => {\r",
+											"                pm.expect(originalEntity[key], key).to.equals(updatedEntity[key]);\r",
+											"            })\r",
+											"    });\r",
+											"\r",
+											"    pm.test(\"Every non-primary key value on the entity should be changed\",\r",
+											"    () => {\r",
+											"        for (const [key, value] of Object.entries(originalEntity)) {\r",
+											"            if (keys.includes(key)) { \r",
+											"                continue;\r",
+											"            }\r",
+											"            console.log(key)\r",
+											"            if (originalEntity[key].length !== updatedEntity[key].length) {\r",
+											"                pm.expect(originalEntity[key], key).to.not.deep.equal(updatedEntity[key]);\r",
+											"            }\r",
```

Comment:
We discussed today, I think we can improve this logic. Watch out for `console.log`